config/rbac/module_loader_scc.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: security.openshift.io/v1
+kind: SecurityContextConstraints
+metadata:
+  name: module-loader
```

Comment:
```suggestion
  name: kmm-operator-module-loader
```

Reply:
This is also resolved via https://github.com/rh-ecosystem-edge/kernel-module-management/pull/53#discussion_r981120263.